mazda_cx5_gt_2017.dbc

```diff
@@ -485,15 +485,15 @@ BO_ 606 SPEED_TBD: 8 XXX
  SG_ NEW_SIGNAL_1 : 23|12@0- (1,0) [0|65535] "" XXX
 
 BO_ 552 GEAR: 8 XXX
- SG_ NEW_SIGNAL_2 : 8|1@0+ (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_3 : 11|1@0+ (1,0) [0|1] "" XXX
- SG_ NEW_SIGNAL_4 : 18|3@0+ (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_5 : 26|3@0+ (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_6 : 31|5@0+ (1,0) [0|31] "" XXX
  SG_ NEW_SIGNAL_7 : 39|1@0+ (1,0) [0|255] "" XXX
- SG_ MORE_GEAR : 7|4@0+ (1,0) [0|255] "" XXX
- SG_ GEAR : 35|3@0+ (1,0) [0|15] "" XXX
- SG_ REVERSE : 36|1@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_4 : 18|3@0+ (1,0) [0|255] "" XXX
+ SG_ NEW_SIGNAL_2 : 8|1@0+ (1,0) [0|255] "" XXX
```

Comment:
if we don't know what these signals are, we can just leave it out

Reply:
sure, dropped!